public/css/index.css

```diff
@@ -23,3 +23,10 @@
 .red {
   background-color: red !important;
 }
+
+#cartCount {
+  border-radius: 100%;
+  background-color: red;
+  right: -8px;
+  position: absolute;
+}
```

Comment:
Did you try Bootstrap badge?

https://getbootstrap.com/docs/5.3/components/badge/#positioned

I think it might give us the same thing with less CSS

Reply:
Yes, I'm using badge. Look at the JS code, it adds the badge class when the cart component is loaded.

Reply:
I'll try to minimize the css more to get the desired output.

Reply:
oh dang i didn't see that. Ok. Somehow i thought the badge thing would be trivially easy. Looks like i was wrong

Reply:
It's fine, don't have to reduce it too much